src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +57,21 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        String email = "rkrylov@mail.com";
+        String password = "qwerty";
+        authenticationService.register(email, password);
+        System.out.println(userService.findByEmail(email).get());
+        try {
+            System.out.println(authenticationService.login(email, password));
+        } catch (AuthenticationException e) {
+            e.printStackTrace();
```

Comment:
```suggestion
            System.out.println("Can`t login user with email " + email);
```

use `e.printStackTrace();` is bad practise, better to use logger or `System.out.println()` here.